scripts/program/build.mjs

```diff
@@ -18,6 +18,6 @@ await Promise.all(
   getProgramFolders().map(async (folder) => {
     const manifestPath = path.join(workingDirectory, folder, 'Cargo.toml');
 
-    await $`cargo-build-sbf --manifest-path ${manifestPath} ${buildArgs}`;
+    await $`cargo-build-sbf --features bpf-entrypoint --manifest-path ${manifestPath} ${buildArgs}`;
```

Comment:
Just adding a note here since we are doing something slightly different on the other programs – probably all programs are different at this point. 😅 We should converge to a similar pattern.

Reply:
Good point, https://github.com/solana-program/counter-shank/issues/3 to address it